io.github.lucasew.regex101.yaml

```
@@ -0,0 +1,33 @@
+id: io.github.lucasew.regex101
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-23.08
+command: io.github.lucasew.regex101
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-documents/regex101.sqlite
+modules:
+  - name: build
+    sources:
+        - type: git
+          url: https://github.com/lucasew/regex101.git
+          tag: 0.3.0
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    post-install:
+      - install -Dm644 -t ${FLATPAK_DEST}/share/applications installer/regex101.desktop
+      - mv ${FLATPAK_DEST}/share/applications/{regex101,io.github.lucasew.regex101}.desktop
+      - install -Dm644 -t ${FLATPAK_DEST}/share/share/metainfo io.github.lucasew.regex101.xml
+      - |
+        for res in 16 32 64 128 256 512; do
+          res="${res}x${res}"
+          item=${FLATPAK_DEST}/share/icons/hicolor/$res/apps
+          mkdir -p $item
+          install -m644 assets/regex101.iconset/icon_$res.png $item/io.github.lucasew.regex101.png
```

```suggestion
          install -Dm644 assets/regex101.iconset/icon_$res.png $item/io.github.lucasew.regex101.png
```

however icons should be installed by the upstream build system

---

I did it with CMake, only added a command to delete the 1024px variation

---

it should not install a 1024 size either, it's against the XDG spec (it's not flatpak)